docs/input-kafka.asciidoc

```diff
@@ -375,10 +374,11 @@ we haven't seen any partition leadership changes to proactively discover any new
   * Value type is <<string,string>>
   * There is no default value for this setting.
 
-The class name of the partition assignment strategy that the client uses to
-distribute partition ownership amongst consumer instances. Maps to
-the Kafka `partition.assignment.strategy` setting, which defaults to
-`org.apache.kafka.clients.consumer.RangeAssignor`.
+The name of the partition assignment strategy that the client uses to distribute
+partition ownership amongst consumer instances, supported options are `range`,
+`round_robin`, `sticky` and `cooperative_sticky`. These map to Kafka's corresponding
+https://kafka.apache.org/24/javadoc/org/apache/kafka/clients/consumer/ConsumerPartitionAssignor.html[`ConsumerPartitionAssignor`]
+implementations.
```

Comment:
> Are there cases in the wild where users provide their own ConsumerPartitionAssignor implementations?

there probably aren't since we do not allow loading custom .jar files. 
in theory, there might be if users add stuff to class-path themselves.

> Do we need to document that the fully-qualified path to a class may still be used?

this also came up during https://github.com/logstash-plugins/logstash-integration-kafka/pull/16 as a `partitioner => ...` option was added with enumerated values (also accepting class-names). we have the custom class name documented but I think we should remove that part: 

> The configuration also allows setting a partitioner class name
e.g. `org.apache.kafka.clients.producer.UniformStickyPartitioner`.

so just like here I am for NOT documenting the possibility to set a class-name (unless we're requested to support loading custom classes).